src/main/java/org/broadinstitute/consent/http/db/mapper/InstitutionMapper.java

```diff
@@ -1,5 +1,6 @@
 package org.broadinstitute.consent.http.db.mapper;
```

Comment:
I think this is the only place it needs to be? I could be wrong, but from my understanding, the mapper is called before any reducers, and the InstitutionWithUsersReducer only adds user-related information, since the mapper already caught the institution's fields.